.circleci/config.yml

```diff
@@ -1213,6 +1213,46 @@ jobs:
           name: "Go mod tidy"
           command: make mod-tidy && git diff --exit-code
 
+  op-service-rethdb-tests:
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
+    resource_class: xlarge
```

Comment:
Why does this need xlarge?

Reply:
Don't think it does necessarily, copied resource allocation from the go tests. Changed it 👍